Cargo.toml

```diff
@@ -31,7 +31,13 @@ schemars = { version = "0.9", features = ["derive"] }
 async-trait = "0.1.88"
 log = "0.4.27"
 reqwest = "0.12.18"
-mcp_client_rs = {version = "0.1.7", optional = true}
+rmcp = { git = "https://github.com/modelcontextprotocol/rust-sdk", branch = "main", features = [
```

Comment:
I do not accept dependencies that are based on not-yet-released code (no linking to Git as a source for dependencies). The only possible exception is that some library was not yet released at all.